---a/net/minecraft/scoreboard/GoalColor.java
+++b/net/minecraft/scoreboard/GoalColor.java
@@ -1,35 +1,32 @@
 package net.minecraft.scoreboard;
 
 import java.util.List;
+
 import net.minecraft.util.EnumChatFormatting;
 
-public class GoalColor implements IScoreObjectiveCriteria
-{
+public class GoalColor implements IScoreObjectiveCriteria {
+
     private final String goalName;
     private static final String __OBFID = "CL_00001961";
 
-    public GoalColor(String p_i45549_1_, EnumChatFormatting p_i45549_2_)
-    {
+    public GoalColor(String p_i45549_1_, EnumChatFormatting p_i45549_2_) {
         this.goalName = p_i45549_1_ + p_i45549_2_.getFriendlyName();
         IScoreObjectiveCriteria.INSTANCES.put(this.goalName, this);
     }
 
-    public String getName()
-    {
+    public String getName() {
         return this.goalName;
     }
 
-    public int func_96635_a(List p_96635_1_)
-    {
+    public int func_96635_a(List p_96635_1_) {
         return 0;
     }
 
-    public boolean isReadOnly()
-    {
+    public boolean isReadOnly() {
         return false;
     }
 
-    public IScoreObjectiveCriteria.EnumRenderType getRenderType()
-    {
+    public IScoreObjectiveCriteria.EnumRenderType getRenderType() {
         return IScoreObjectiveCriteria.EnumRenderType.INTEGER;
     }
-}
+}
